modin/engines/base/frame/partition_manager.py

```diff
@@ -206,27 +212,45 @@ def groupby_reduce(
             Map function.
         reduce_func : callable,
             Reduce function.
-        apply_indices : list of ints, default: None
-            Indices of `axis ^ 1` to apply function over.
+        selection : dict[int] -> list of labels, default: None
+            Set of indices/columns to apply aggregation on, where the dictionary indices
+            are numeric indices of partitions to apply and the values are the indices/columns
+            to apply in this particular partitions.
 
         Returns
         -------
         NumPy array
             Partitions with applied groupby.
         """
-        if apply_indices is not None:
+        reduce_func_proxy = reduce_func
+        if selection is not None:
+            parts_to_select = list(selection.keys())
```

Comment:
a comment should be added on the logic in this if statement and probably the difference between `parts_to_select` and `partitions_selections` if it is not clear from the logic being described in the comment.

Reply:
it's described at the `selection` docstring:
- `parts_to_select` are partition indices to apply function on (because it's `selection.keys()`)
- `partitions_selections` are selections for every selected partitions (because it's `selection.values()`)